styles.css

```diff
@@ -647,10 +665,265 @@ form textarea ::placeholder {
 }
 
-@media only screen and (min-width: 375px) {
+@media only screen and (max-width: 767px) {
+  .menu {
+    display: none;
+  }
 }
 
 /* minimum desktop screen size : 768px */
 @media only screen and (min-width: 768px) {
-
-}
\ No newline at end of file
+  .page-1_container {
+    margin: auto;
+  }
+
+  header {
+    margin-bottom: -5%;
+    margin-top: -5%;
+  }
+
+  .my-logo {
+    margin-right: 70%;
+    margin-top: 6.5%;
+  }
+
+  .collapsed_menu {
+    display: none;
+  }
+
+  .menu {
+    margin-top: 3%;
+  }
+
+  .headline {
+    width: 100%;
+    background-image: url('images/Header\ bg.png');
+    background-position: fixed;
+    background-size: 100% 80%;
+    padding-bottom: 8%;
+  }
+
+  .headline h1 {
+    margin-left: 17%;
+    margin-bottom: 7%;
+    margin-top: 10%;
+    padding-top: 15%;
+  }
+
+  .headline p {
+    width: 50%;
+    margin-left: 17%;
+  }
+
+  .headline div {
+    margin-left: 17%;
+    margin-top: -3%;
+  }
+
+  .headline ul {
+    margin-left: 17%;
+    margin-top: -2%;
+    margin-bottom: 10%;
+    gap: 2%;
+  }
+
+  .card_works {
+    box-sizing: border-box;
+    display: grid;
+    width: 1156px;
+    height: 476px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 3%;
+  }
+
+  .snapshot_portfolio {
+    float: left;
+    width: 544px;
+    height: 448px;
+    background-size: contain;
+    padding: 0;
+    margin: 0;
+  }
+
+  .snapshot_portfolio_1 {
+    float: right;
+    width: 544px;
+    height: 448px;
+    background-size: contain;
+    padding: 0;
+    margin: 0;
+  }
+
+  .tags {
+    width: 15%;
+  }
+
+  .tags div {
+    width: fit-content;
+    block-size: fit-content;
+    white-space: nowrap;
+  }
+
+  .left_block {
+    width: 50%;
+    margin-left: 54%;
+    margin-top: -85%;
+  }
+
+  .work_1 .snapshot_portfolio {
+    background: url('images/Snapshoot\ Portfolio.png');
+    margin-top: -4%;
+    margin-left: -4%;
+  }
+
+  .work_2 .snapshot_portfolio_1 {
+    margin-left: 50%;
+  }
+
+  .work_2 .left_block {
+    margin-left: -2%;
+    margin-top: -92%;
+  }
+
+  .work_3 .snapshot_portfolio {
+    margin-left: -4%;
+    margin-top: -3%;
+  }
+
+  .work_3 .see_project {
+    background-color: #6070ff;
+    color: white;
+  }
+
+  .work_4 .snapshot_portfolio_1 {
+    margin-left: 50%;
+    background: url('images/uber.png');
+    margin-top: -4%;
+  }
+
+  .work_4 .left_block {
+    margin-left: -2%;
+  }
+
+  .card_works + .card_works {
+    margin-top: 10%;
+  }
+
+  .works {
+    margin-left: 10%;
+  }
+
+  .about_header {
+    margin-bottom: -0.1%;
+  }
+
+  .about_description {
+    margin-left: -58%;
+    margin-top: 0.99%;
+    width: 447px;
+  }
+
+  .connect {
+    margin-top: -2%;
+    margin-left: 14%;
+  }
+
+  .about_myself ul {
+    gap: 1%;
+    margin-top: 1%;
+    margin-left: 1%;
+  }
+
+  .resume_button {
+    width: 9.65%;
+    margin-left: -76%;
+    margin-top: 1%;
+    margin-right: 0;
+    padding: 0.7%;
+    border-radius: 16px;
+  }
```

Comment:
- [ ] As suggested by the previous reviewer, the `Get me resume` button does not look exactly as we have it on the figma design. You need to adjust the `border-radius` value in order to match with the figma design.

### Your Design
![your-get-resume-btn-mobile](https://user-images.githubusercontent.com/26597432/221378389-9b0f637f-624b-4f9d-8552-c1a95fc6226a.png)

### Figma Design
![temp-get-resume-btn-mobile](https://user-images.githubusercontent.com/26597432/221378477-ac8784aa-247d-488f-aa14-1678e7d67c76.png)